### PR TITLE
docs: rename catapult-server to catapult-client in docs

### PR DESCRIPTION
**Pull request by Jaguar0625**

### CODE REVIEW

**Review by Wayonb — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# catapult-server
+# catapult-client
 
 [![docs](badges/docs--green.svg)](https://nemtech.github.io)
```
Comment (segfaultxavi) [inline README.md:3]:
```suggestion
[![docs](badges/docs--green.svg)](https://symbol.github.io)
```

File: README.md [hunk 2]
```diff
@@ -7,17 +7,17 @@
 
 Symbol-based networks rely on nodes to provide a trustless, high-performance, and secure blockchain platform.
 
-These nodes are deployed using [catapult-server] software, a C++ rewrite of the previous Java-written [NEM] distributed ledger that has been running since 2015.
+These nodes are deployed using [catapult-client] software, a C++ rewrite of the previous Java-written [NEM] distributed ledger that has been running since 2015.
 
 Learn more about the protocol by reading the [whitepaper] and the  [developer documentation].
```
Comment (segfaultxavi) [inline README.md:12]:
```suggestion
Learn more about the protocol by reading the [technical reference] and the  [developer documentation].
```

File: README.md [hunk 3]
```diff
@@ -7,17 +7,17 @@
...
 ## Package Organization
 
-catapult-server code is organized as follows:
+catapult-client code is organized as follows:
 
 | Folder name | Description |
 | -------------|--------------|
-| /extensions | Modules that add features to the bare catapult-server. These capabilities are all optional because none of them impact consensus. |
+| /extensions | Modules that add features to the bare catapult-client. These capabilities are all optional because none of them impact consensus. |
 | /external | External dependencies that are built with the server. |
```
Comment (segfaultxavi) [inline README.md:21]:
```suggestion
| /external | External dependencies that are built with the client. |
```

File: README.md [hunk 4]
```diff
@@ -82,10 +82,10 @@ Copyright (c) 2018 Jaguar0625, gimre, BloodyRookie, Tech Bureau, Corp Licensed u
...
-[issues]: https://github.com/nemtech/catapult-server/issues
+[issues]: https://github.com/symbol/catapult-client/issues
 [slack]: https://join.slack.com/t/nem2/shared_invite/enQtMzY4MDc2NTg0ODgyLWZmZWRiMjViYTVhZjEzOTA0MzUyMTA1NTA5OWQ0MWUzNTA4NjM5OTJhOGViOTBhNjkxYWVhMWRiZDRkOTE0YmU
-[catapult-server]: https://github.com/nemtech/catapult-server
+[catapult-client]: https://github.com/symbol/catapult-client
 [symbol-rest]: https://github.com/nemtech/catapult-rest
-[Service Bootstrap]: https://github.com/tech-bureau/catapult-service-bootstrap
+[Service Bootstrap]: https://github.com/symbol/symbol-bootstrap
 [nem]: https://nem.io
 [whitepaper]: https://nemtech.github.io/catapult-whitepaper/main.pdf
```
Comment (segfaultxavi) [inline README.md:91]:
```suggestion
[technical reference]: https://symbol.github.io/symbol-technicalref/main.pdf
```

File: docs/RUNPEERLIN.md [hunk 5]
```diff
@@ -1,10 +1,10 @@
 # Running a Peer Node on Ubuntu 18.04 (LTS)
 
-These instructions summarize the minimum number of steps to run a [peer node](https://nemtech.github.io/concepts/node.html#id1) and connect it to an existing network using the catapult-server build.
+These instructions summarize the minimum number of steps to run a [peer node](https://nemtech.github.io/concepts/node.html#id1) and connect it to an existing network using the catapult-client build.
```
Comment (segfaultxavi) [inline docs/RUNPEERLIN.md:3]:
```suggestion
These instructions summarize the minimum number of steps to run a [peer node](https://symbol.github.io/concepts/node.html#id1) and connect it to an existing network using the catapult-client build.
```

**Review by segfaultxavi — CHANGES_REQUESTED** [review]
There are more broken links but I will submit another patch for them as they are unrelated to this changeset.

**Review by gimer — APPROVED** [review]

**Review by segfaultxavi — APPROVED** [review]

**Review by Wayonb — APPROVED** [review]

**Pull request merged**